nft/handler.go

```diff
@@ -36,3 +43,17 @@ func (g grpcHandler) MintNFT(context context.Context, request *nftpb.NFTMintRequ
 	watchToken := <-confirmation
 	return &nftpb.NFTMintResponse{TokenId: watchToken.TokenID.String()}, watchToken.Err
 }
+
+func (g grpcHandler) validateParameters(request *nftpb.NFTMintRequest) error {
+
+	if !common.IsHexAddress(request.RegistryAddress) {
```

Comment:
Usage of these addresses part of next PR?

Reply:
yes